.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,112 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["kind/bug"]
```

Comment:
Should Bugs also be labeled as untriaged?

Reply:
...or triage? 
ref link: https://learn.wordpress.org/tutorial/how-to-do-triage-on-github/